app/src/main/kotlin/com/softteco/template/ui/feature/resetPassword/ResetPasswordScreen.kt

```diff
@@ -68,13 +70,15 @@ private fun ScreenContent(
     ) {
         Column(
             modifier = Modifier
-                .padding(Dimens.PaddingExtraLarge),
+                .padding(Dimens.PaddingExtraLarge)
+                .fillMaxSize(),
             verticalArrangement = Arrangement.spacedBy(Dimens.PaddingExtraLarge),
             horizontalAlignment = Alignment.CenterHorizontally
         ) {
-            if (state.loading) {
-                Text(stringResource(id = R.string.loading))
-            }
+            Text(
```

Comment:
What are the advantages of this approach?
The previous version is shorter, more readable, and doesn't add hidden components to a screen.

Reply:
To avoid text jumping I made a component visibility View. Gone or View.Visible.